_component_code/choice-tiles.md

```diff
@@ -2,7 +2,7 @@
 title: Option Tiles
 layout: code
 category: Components Code
-code: components/option-tiles
+code: components/option-tile
```

Comment:
This would still 404, because we haven't renamed it yet (currently, it's `components/choice-tile`)

Reply:
updated this to `code: components/choice-tile`